libs/langchain/langchain/output_parsers/xml.py

```diff
@@ -0,0 +1,34 @@
+import re
+from typing import List, Optional
+
+from langchain.output_parsers.format_instructions import XML_FORMAT_INSTRUCTIONS
+from langchain.schema import BaseOutputParser
+
+
+class XMLOutputParser(BaseOutputParser):
+    """Parse an output using xml format."""
+
+    tags: Optional[List[str]] = None
+
+    def get_format_instructions(self) -> str:
+        if self.tags:
+            return XML_FORMAT_INSTRUCTIONS.format(tags=self.tags)
+        return """Write a following string formatted as an XML file.
+        Always mention the encoding of the file in the first line.
```

Comment:
I am not sure we care about encoding, it is verbose and generates additional costs.